src/AFQMC/Matrix/csr_matrix_construct.hpp

```diff
@@ -70,11 +70,11 @@ CSR construct_csr_matrix_single_input(MultiArray2D&& M, double cutoff, char TA,
     }
     else
     {
-      nr = M.size(1);
-      nc = M.size(0);
+      nr = std::get<1>(M.sizes());
+      nc = std::get<1>(M.sizes());
```

Comment:
suspicious change.

Reply:
yes, thank you for (train)spotting this.